### ISSUE TITLE
Ray not sending integers

### ISSUE THREAD

**Issue by nlemsieh** [spatie/ray#379]
**Describe the bug:**
I'm using Ray in WordPress and framework-agnostic PHP projects and for some reason integers don't show up in Ray app.

**Versions:**
Ray version:  1.14.5
PHP version: 7.4.13
Ray WP plugin: 1.2.4

**To Reproduce:**
Try sending an integer value, i.e.:
```php
ray(25);
```

**Desktop:**
 - OS: macOS
 - Version 10.13.6

**Comment by lorlab** [comment]
I'm also noticing this in Laravel

**Comment by SpencerCloud** [comment]
Posting this as a potential workaround for people before this gets fixed. It's simple but took a little too long to occur to me to do this. Just typecast any integers to a string when outputting in the debugger.

`ray((string) 25);`

**Comment by Muffinman** [comment]
Can confirm I'm also seeing this on Drupal.

PHP Library version: `1.21.2`
Ray client: `1.14.5`
PHP: 8.0.3
MacOS

**Comment by tomcoonen** [comment]
Also having this issue, even when multiple arguments are fed to the `ray()` function, as soon as an integer is include all these are not showing up.

**Comment by ajimatahari** [comment]
Hapenned to me as well

**Comment by vicenterusso** [comment]
+1. Using Laravel + Linux (ubuntu)

Terminal Log:
```
TypeError: e.includes is not a function
    at D (/tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:367:19406)
    at /tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:367:18468
    at Array.map (<anonymous>)
    at /tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:367:18444
    at Array.map (<anonymous>)
    at /tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:367:18198
    at s.handle_request (/tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:156:784)
    at s (/tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:149:883)
    at u.dispatch (/tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:149:905)
    at s.handle_request (/tmp/.mount_Ray-1.UH5UqJ/resources/app.asar/dist/main.js:156:784)
```

**Comment by nnerijuss** [comment]
Have same issue with M1 mac. Until lates update everything works just fine.

**Comment by AdrianMrn** [comment]
Thanks for the reports and especially thanks too @vicenterusso for the logs which gave me an idea about what could be causing the bug :)

This has been fixed and will be included in the 1.14.6 patch version (releasing now).
